packages/main-menu-react/src/contexts/mainMenuContext.tsx

```diff
@@ -1,19 +1,27 @@
 import { createContext, useContext } from "react";
 
 export type NavigationFunction = (path: string) => void;
-export type IsActiveFunction = (path: string) => boolean;
+export type IsActiveFunction = (path: string, currentOpenMenu: string) => boolean;
 interface MainMenuContextProps {
     navigationFunction: NavigationFunction;
     isActiveFunction: IsActiveFunction;
+    reduceMotion: boolean;
 }
 
 export const defaultNavigationFunction = (path: string) => {
     window && window.location.assign(path);
 };
+export const defaultIsActiveFunction = (path: string, currentOpenMenu: string) => {
+    const thisMenuIsOpen = path !== "" && currentOpenMenu.includes(path);
+    const itemPathMatches = window.location ? window.location.pathname.includes(path) : false;
+
+    return thisMenuIsOpen || (itemPathMatches && (thisMenuIsOpen || currentOpenMenu == ""));
```

Comment:
Please split this
![image](https://user-images.githubusercontent.com/46530368/94123575-42d22e00-fe54-11ea-8d59-a3cac7e7f14d.png)